### PR TITLE
Improved duplicate linking

### PR DESCRIPTION
**Pull request by VSEphpbb**
Converts from the old and cumbersome way of linking duplicate by their Idea ID#:
duplicateold

To using the Idea Titles using live search with immediate clickable results:
DuplicateLinks

### CODE REVIEW

File: factory/ideas.php [hunk 1]
```diff
@@ -416,6 +444,25 @@ public function set_title($idea_id, $title)
...
+	 * @param int $id ID of an idea
+	 *
+	 * @return string The idea's title
+	 */
+	public function get_title($id)
+	{
+		$sql = 'SELECT idea_title
+			FROM ' . $this->table_ideas . '
+			WHERE idea_id = ' . (int) $id;
+		$result = $this->db->sql_query_limit($sql, 1);
```
Comment (DavidIQ) [inline factory/ideas.php:459]: Can we add some caching here? Maybe 300 or 600 seconds (5/10 minutes)? Not sure what's a good caching time for this.

Comment (VSEphpbb) [reply factory/ideas.php:459]: Is that worth it? because a cache purge action will then need to be added as well whenever soemone edits a title.

Comment (DavidIQ) [reply factory/ideas.php:459]: Maybe not. Table probably won't be huge. We can revisit this if we run into performance issues.

File: language/en/common.php [hunk 2]
```diff
@@ -78,7 +78,7 @@
 
 	'TICKET'				=> 'Ticket',
 	'TICKET_ERROR'			=> 'Ticket ID must be of the format “PHPBB3-#####”.',
-	'TICKET_ERROR_DUP'		=> 'Please use the Idea ID number.',
+	'TICKET_ERROR_DUP'		=> 'You must click on an idea title from the live search results. To delete the duplicate, clear the field and hit ENTER. To exit this field click ESC.',
```
Comment (DavidIQ) [inline language/en/common.php:81]:
```suggestion
	'TICKET_ERROR_DUP'		=> 'You must click on an idea title from the live search results. To delete the duplicate, clear the field and press ENTER. To exit this field press ESC.',
```

**Pull request merged**